### PR TITLE
Docs.ap/reorg for two groups

### PR DESCRIPTION
**Pull request by roedoejet**
just a PR for a rebased version of [docs.reorg-for-two-groups](https://github.com/MotherTongues/mothertongues/tree/docs.reorg-for-two-groups)

@pandersity, I wonder why it doesn't show your avatar. It looks like it still says "PT Anderson" authored, but it's weird it doesn't show your avatar...

### CODE REVIEW

**Comment by pandersity** [comment]
@roedoejet , this is ready this ready for review

File: docs/index.md [hunk 1]
```diff
@@ -39,28 +39,27 @@ Each front end tool must accept the Mother Tongues data format. The schema for t
...
-2.  Write a valid Mother Tongues Language configuration file. See
-    [guides](guides/index.md)
-3.  Write valid Mother Tongues data resource configuration files. See
-    [guides](guides/index.md)
+### Option 1: No-Code Option (Fastest)
+If any of the following describe you, then the no-code path may be right for you:
 
-Then, build with the command line:
+- I want my dictionary running as quickly as possible with the least lift
+- I want to try out-of-the-box MTD "as-is" (in other words, you don't need to make major code customizations right now)
```
Comment (roedoejet) [inline docs/index.md:48]: maybe add "I'm not as comfortable with coding" or something similar?

**Review by roedoejet — COMMENTED** [review]
This is awesome @pandersity ! I really like the reorganization. I've made a couple small comments, questions, and suggestions, I'll be happy to get your thoughts. At the same time, I'll try and add the no-code guides soon and then merge this in.

File: docs/guides/search.md [hunk 2]
```diff
@@ -133,9 +145,11 @@ Stemming is all about normalizing words by turning them into the 'stem' (version
 }
 ```
 
+If you would to use other stemmers (for example, if the L2 is in Spanish, Arabic, etc.) - see these documents... TBD
+
 ## Punctuation
 
-You might want to remove all punctuation from your search results, in case you want to return a result for `"hello"` when someone searches for `hello`, for example.
+You might want to remove all punctuation from your search results, in case you want to return a result for `"hello"` when someone searches for `h,ello`, for example.
```
[the remark this answers is not in the document]
Comment (pandersity) [reply docs/guides/search.md:152]: hm, yeah, I think maybe compound word might make the example clearer. Will float another option

File: docs/developer/unittests.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# MotherTongues Unit Tests
+
+Unit tests for the MotherTongue repo are found in `/mothertongues/tests/`
+
+Test runner is found in `run.py` script.
+
+## Install Packages
+Run the following command
+
+`poetry run pip install g2p`
```
[the remark this answers is not in the document]
Comment (pandersity) [reply docs/developer/unittests.md:15]: ah yes, you are correct, will remove that

File: docs/index.md (hunk 1 above)
Comment (pandersity) [reply docs/index.md:49]: added

File: docs/guides/prepare.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ comments: true
 
 # Preparing your data
 
-*These guides assume you are comfortable with the [Command Line](https://en.wikipedia.org/wiki/Command-line_interface), [Git](https://en.wikipedia.org/wiki/Git) and [Python](https://en.wikipedia.org/wiki/Python_(programming_language)). You must have all of these installed on your machine. You are also strongly encouraged to have a [GitHub](https://github.com) account. You are encouraged to fork or clone the [Mother Tongues Dictionary Starter](https://github.com/MotherTongues/mtd-starter) and follow along.*
+*This guide is required for anyone creating a Mother Tongues Dictionary, regardless of which path they take (no code or advanced developer).*
```
[the remark this answers is not in the document]
Comment (pandersity) [reply docs/guides/prepare.md]: done!

File: docs/guides/search.md [hunk 5]
```diff
@@ -3,9 +3,16 @@ comments: true
 ---
 
 # Approximate Search
+*This guide applies for all paths (no-code and advanced developer)*
```
[the remark this answers is not in the document]
Comment (pandersity) [reply docs/guides/search.md]: done

**Pull request merged**